### PR TITLE
Update 03_Configuration.md

### PR DESCRIPTION
**Pull request by kaurov**
param injection sample

  
  

## Changes in this pull request  
Resolves #

### <samp>🤖 Generated by Copilot at c4aeff7</samp>

Added documentation for injecting parameters from `.yml` files to class variables. The change shows how to define and use a service with a parameter argument.

### <samp>🤖 Generated by Copilot at c4aeff7</samp>

> _Inject parameters_
> _From `.yml` to service_
> _Springtime of coding_

### <samp>🤖 Generated by Copilot at c4aeff7</samp>

*  Add a new subsection to the documentation about injecting parameters from the .yml file to a class variable ([link](https://github.com/pimcore/pimcore/pull/15020/files?diff=unified&w=0#diff-4f89461e9826383b48247c5b15ee3b299149fe4781ca99f4dc94e2d55a36ed44R107-R125))

### CODE REVIEW

File: doc/01_Getting_Started/03_Configuration.md [hunk 1]
```diff
@@ -104,3 +104,22 @@ use \Symfony\Component\HttpFoundation\Request;
...
+            
+
+Then in your class you can access this param in **2 approaches**:
+
+    public function __construct(string $emails, private ParameterBagInterface $params)
+    {
+        $emailsArray =  $this->params->get('email.debug.email_addresses');
+        var_dump($emails, $emailsArray);
+    }
+
```
Comment (jdreesen) [inline doc/01_Getting_Started/03_Configuration.md]:
````suggestion
## How to inject the .yaml param to class variable

In your `services.yaml` add your service, e.g.:

```yaml
    SomeBundle\Service\SomeSender:
        arguments:
            $emails: '%email.debug.email_addresses%'
```

Then in your class you can access this param in **2 approaches**:

```php
    public function __construct(string $emails, private ParameterBagInterface $params)
    {
        $emailsArray =  $this->params->get('email.debug.email_addresses');
        var_dump($emails, $emailsArray);
    }
```
````

**Comment by fashxp** [comment]
that is pretty symfony standard and nothing specific to Pimcore, right? 
I would prefer to link to symfony docs rather than copying it.

**Comment by mcassier31** [comment]
As @fashxp mentioned it, this is Symfony specific. Please replace the code mention with https://symfony.com/doc/current/service_container.html#manually-wiring-arguments

**Comment by fashxp** [comment]
I'll close this here, feel free to create another PR with link to symfony docs. Thank you very much.

**Pull request closed without merging**